client/src/List.css

```diff
@@ -0,0 +1,28 @@
+.list-container {
```

Comment:
Sorry, should have mentioned this in the kickoff meeting. We want to keep all of the styling within a makeStyles function in the component rather than a css file

Reply:
I switched it over now